site/docs/v1/tech/apis/identity-providers/_links-post-response-body.adoc

```diff
@@ -11,10 +13,19 @@ The unique Id of the identity provider.
 The Id for the User that is provided by the identity provider.
 
 [field]#identityProviderLink.userId# [type]#[UUID]#::
-The FusionAuth User Id that is linked to the identity provider.
+The unique Id of the FusionAuth User that is being linked to the identity provider.
+
+[field]#identityProviderLink.insertInstant# [type]#[Long]#::
+The link:/docs/v1/tech/reference/data-types#instants[instant] the Link was added to the FusionAuth database.
+
+[field]#identityProviderLink.lastLoginInstant# [type]#[Long]#::
+The last link:/docs/v1/tech/reference/data-types#instants[instant] of the User's most recent login to the identity provider linked Application.
+
+[field]#identityProviderLink.tenantId# [type]#[UUID]#::
+The Id of the Tenant that this User belongs to.
 
 [source,json]
 .Example Response JSON
 ----
 include::../../../../src/json/identity-providers/links/create-response.json[]
```

Comment:
How is this different than `response.json` which also as a `token` property in the response?